src/cli.rs

```diff
@@ -351,6 +370,25 @@ pub struct InspectSnapshot {
     /// Read the snapshot raw bytes directly, with no base64 parsing
     #[clap(long)]
     pub raw_input: bool,
+    /// Query the snapshot
+    #[clap(
+        long,
+        value_parser = clap::builder::ValueParser::new(parse_rule),
+    )]
+    pub query: Option<Rule>,
+    /// Query facts from all blocks (not just authority, authorizer or explicitly trusted blocks). Be careful, this can return untrustworthy facts.
```

Comment:
Why could it return untrustworthy facts?

Reply:
it can return facts from attenuation blocks, which are not signed with a well-known keypair, so they can be freely added by the token holder.